docs/applying-terraform.md

```diff
@@ -66,3 +66,57 @@ aws ecs wait services-stable \
  --services "$APPLICATION" \
  --region "$AWS_REGION"
 ```
+
+### Monitoring
+
+The monitoring stack of GOV.UK is in a separate cluster compared to the one where
+the GOV.UK apps run. Below are the details how to run deploy the monitoring stack
+which includes only Grafana for now.
+
```

Comment:
It might also be worth mentioning in the docs that if we are applying to a new environment, we'll need to setup a GitHub app (and manually add `github_client_secret` to SecretsManager).

Reply:
good point. I will do that.